doc/classes/JSONParseResult.xml

```diff
@@ -68,12 +70,24 @@
 	</methods>
 	<members>
 		<member name="error" type="int" setter="set_error" getter="get_error" enum="Error">
+			Returns error's type if JSON source not successfull parsed. See [@Global Scope].Error.
```

Comment:
In general, member descriptions don't say "Returns". This is a value, not a function, and it usually can both set and get.
Try:
"The error type if if JSON source was not successfully parsed." 

That link at the end isn't right.  I don't think we have constant links, so you have to say something like "See [@Global Scope] ERR_* constants."